### PR TITLE
focus@will.focusatwill version 1.6.0

### PR DESCRIPTION
**Pull request by AndreaTheLinguist**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56736)

### CODE REVIEW

File: manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: focus@will.focusatwill
+PackageVersion: 1.6.0.44
+PackageLocale: en-US
+Publisher: focus@will
+PackageName: focusatwill
+License: Copyright © 2019 focus@will
```
Comment (Trenly) [inline manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.locale.en-US.yaml:9]: A copyright is no a license; Please provide the actual package license terms such as `Proprietary`, `MIT`, `GPL-2.0` or whatever package license terms are applicable

File: manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: focus@will.focusatwill
+PackageVersion: 1.6.0.44
```
Comment (Trenly) [inline manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.locale.en-US.yaml:5]: This must match what is shown in control panel
```suggestion
PackageVersion: 1.6.0
```

File: manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: focus@will.focusatwill
+PackageVersion: 1.6.0.44
```
Comment (Trenly) [inline manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.installer.yaml:5]: This must match what is shown in control panel
```suggestion
PackageVersion: 1.6.0
```

File: manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: focus@will.focusatwill
+PackageVersion: 1.6.0.44
```
Comment (Trenly) [inline manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.yaml:5]: This must match what is shown in control panel
```suggestion
PackageVersion: 1.6.0
```

**Review by Trenly — CHANGES_REQUESTED** [review]
When updating the version, don't forget to update the name of the folder the manifest is in which corresponds to the version number

File: manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.locale.en-US.yaml (hunk 1 above)
Comment (AndreaTheLinguist) [reply manifests/f/focus@will/focusatwill/1.6.0.44/focus@will.focusatwill.locale.en-US.yaml:9]: ok, so... I don't know what that means... 😬

**Pull request closed without merging**